src/Http/Productions/ProductionSuggestionController.php

```diff
@@ -34,13 +34,13 @@ public function nextSuggestion(): Response
             $date = new \DateTime();
             $date->setTime(0, 0, 1);
             $suggestion = $this->similaritiesClient->nextSuggestion($date);
-            $eventTwo = $this->enrichedEventRepository->get($suggestion->getEventOne());
             $eventOne = $this->enrichedEventRepository->get($suggestion->getEventTwo());
+            $eventTwo = $this->enrichedEventRepository->get($suggestion->getEventOne());
```

Comment:
There's still a minor issue here, which was maybe not clear from my first comment:
The variable name is `$eventTwo` but it is based on `$suggestion->getEventOne()`. Vice-versa for `$eventOne` above.

Reply:
The rest of the changes look good, almost there! 👌

Reply:
Resolved and added some extra cleanup